tsdfileapi/api.py

```diff
@@ -1958,7 +1958,11 @@ def delete(self, tenant, filename):
                 self.message = 'Cannot perform DELETE on directory - delete files individually'
                 raise Exception
             try:
+                # Allow the file to be deleted by changing the rights temporary of the parent directory
+                subprocess.call(['sudo', 'chmod', 'o+w',  os.path.dirname(self.filepath)])
```

Comment:
Actually, it occurred to me that we only need to do this, and the reverse, when `if self.has_posix_ownership:`.